### ISSUE TITLE
RCS1021 doesn't trigger on statements broken over multiple lines

### ISSUE THREAD

**Issue by IanKemp** [JosefPihrt/Roslynator#196]
Extremely contrived example of a case that isn't handled:

```csharp
var things = new List<string>() { "foo", "bar", "baz" };
var barIndex = things.FindIndex(item =>
{
    // no RCS1021
    return item.IndexOf("bar")
        != -1;
});
```

and one that is:

```csharp
var things = new List<string>() { "foo", "bar", "baz" };
var barIndex = things.FindIndex(item =>
{
    // RCS1021
    return item.IndexOf("bar") != -1;
});
```

Note that the only difference is the arbitrary newline inserted after the comparison operator; lexically the code is the same in both cases.

**Comment by JosefPihrt** [comment]
RCS1021 intentionally works only for single-line expression.

Purpose of this analyzer is not to save two lines by removing braces. Purpose is to remove braces and return keyword when they are clearly unnecessary which is the case of single-line expression.

When the expression is multi-line it is up to developer how the lambda should look like.
